### PR TITLE
Switched out Laravel Mix for Rollup

### LINKED ISSUE #11: Rollup / micro bundle over Webpack
I think we can get some smaller bundle / extension sizes by switching to a pure Rollup bundler or Microbundle.

### PR DESCRIPTION
**Pull request by ryangjchandler**
Closes #11. This PR switches out Laravel Mix in favour of Rollup. The main reason being a smaller extension size, since Laravel Mix passed JavaScript through `vue-template-compiler` which can result in unnecessary polyfills or code adjustments being made.

Running it through Rollup simply minifies and resolves any imports such as Alpine itself. After building and outputting to the `dist/chrome` folder, the overall ZIP'd size of the extension is roughly 15-20kb smaller than with Mix.

I've also moved the CSS to an import statement in the `panel.js` file. This was the simplest way for Rollup to process it. It will automatically get included in the `<head>` of the panel's HTML.

Let me know if you spot anything obvious that I've missed but testing it on Chrome locally, everything works as expected. @HugoDF I'm not sure if you want to checkout the branch and test it in Firefox since you made the changes to support it, I've got it running in Developer Edition fine at the moment so can't see any reason why it wouldn't work.

As a final note, I've not minified the `proxy.js` file on build since minification of this file could break the extension with duplicate variable names / redeclarations.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-  }
+    "name": "alpinejs-devtools",
+    "version": "0.0.1",
+    "description": "Simple Devtools for Alpine.js To Make Your Life Simpler",
+    "scripts": {
+        "build": "rollup -c",
+        "watch": "rollup -c -w"
+    },
+    "author": "",
+    "license": "ISC",
```
Comment (HugoDF) [inline package.json:10]: Should probably swap this to MIT

Comment (ryangjchandler) [reply package.json:10]: Yeah good shout.

**Comment by ryangjchandler** [comment]
> What about instructions in the README?

Will do this alongside MIT.

**Comment by ryangjchandler** [comment]
Good spot. Will need to add the correct RegEx to PurgeCSS.

File: postcss.config.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    plugins: [
+        require('tailwindcss'),
+        require('@fullhuman/postcss-purgecss')({
```
Comment (HugoDF) [inline postcss.config.js:4]: Should we skip this on dev builds? Could be a simple `process.env.NODE_ENV !== production`

Unless rollup will recompile the CSS on any HTML change?

**Comment by HugoDF** [comment]
Kind of unrelated comment, do we need an `.editorconfig`? I'm weary of defaulting to 2 space tabs vs this project where it's 4

**Comment by ryangjchandler** [comment]
> Kind of unrelated comment, do we need an `.editorconfig`? I'm weary of defaulting to 2 space tabs vs this project where it's 4

Yeah, I was going to add one in this branch but wasn't sure what everyone's preference was. I'm a 4 space indent guy, but that's coming from Laravel so whatever floats your boat.

**Comment by Te7a-Houdini** [comment]
> > Kind of unrelated comment, do we need an `.editorconfig`? I'm weary of defaulting to 2 space tabs vs this project where it's 4
> 
> Yeah, I was going to add one in this branch but wasn't sure what everyone's preference was. I'm a 4 space indent guy, but that's coming from Laravel so whatever floats your boat.

for me am ok with whatever you choose guys .

**Comment by ryangjchandler** [comment]
@HugoDF what's your usual preference?

**Comment by HugoDF** [comment]
@ryangjchandler no preference as long as there's an editorconfig 😁

**Comment by ryangjchandler** [comment]
Okay, @Te7a-Houdini @HugoDF shall we say 4 spaces for indentation then, I'll add an editorconfig to this branch

**Comment by ryangjchandler** [comment]
Sorry guys, I'll make these changes this evening so we can get this PR merged in. It's been a manic couple of days!

**Comment by HugoDF** [comment]
@ryangjchandler no rush mate, hope you're keeping well 🙂

**Comment by ryangjchandler** [comment]
> @ryangjchandler no rush mate, hope you're keeping well 🙂

Yeah, you too mate. Work has been crazy so most of my contributing time has come after 8pm haha.

**Comment by ryangjchandler** [comment]
Need to update the README with new instructions, then everything should be good :)

**Comment by ryangjchandler** [comment]
Didn't need to change the instructions in the README. Are we both cool to merge this? @Te7a-Houdini @HugoDF

**Comment by HugoDF** [comment]
I'm happy with it 🙂

**Comment by Te7a-Houdini** [comment]
same here . everything is working fine on my local chrome

**Pull request merged**